app.go

```diff
@@ -87,6 +89,7 @@ func NewExposerApp(init *ExposerAppInit, ingressClass string, cs kubernetes.Inte
 	}
 
 	app.router.GET("/", app.Greeting).Name = "greeting"
+	app.router.Static("/docs", "./docs")
```

Comment:
I added a step to the Dockerfile to generate the swagger.json file, so it's at least somewhat automated now. I'm going to keep the weird way I included the swagger UI for now since I really like the interactivity of it and being able to make calls to the API directly through it.